### PR TITLE
performatic changes

### PR DESCRIPTION
**Pull request by jrgtt**
I've made some changes in the main.js aiming for a better perfomance in the page load.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,13 +4,6 @@
   <meta charset="UTF-8">
   <title>Who Am I?</title>
   <link rel="stylesheet" href="css/main.css" />
-
-  <script src="js/vendor/jquery.min.js"></script>
-  <script src="js/vendor/lodash.js"></script>
-  <script src="js/vendor/raphael-min.js"></script>
-  <script src="js/vendor/g.raphael-min.js"></script>
-  <script src="js/vendor/piechart.js"></script>
-  <script src="js/main.js"></script>
```
Comment (waldyrious) [inline index.html:13]: Would loading them with the `async` attribute work here?

Comment (jrgtt) [inline index.html:13]: I don't think that would be the case, the page relies on the main.js to write most of it's content and each file must be loaded in order to work.

**Comment by samdoiron** [comment]
To what extent were the changes successful?

**Comment by jrgtt** [comment]
I didn't understand your question.

**Comment by samdoiron** [comment]
Do you have any numbers indicating an improvement? To what extent is it more efficient?

**Comment by jrgtt** [comment]
Not a serious one, it's hard to measure async methods performance, it would require a test library.

For the sake of my consciousness, I put a `var time = new Date().getTime()` before the "squares creating" block. After the loop was done I logged `console.log(new Date().getTime() - time)` the difference was around 500ms.

**Comment by samdoiron** [comment]
That sounds good, and the performance tweaks are reasonable. Thanks for the improvements!

**Pull request merged**